docs/rhel4edge_iso.md

```diff
@@ -66,6 +67,31 @@ The script performs the following tasks:
 
 The artifact of the build is the `scripts/image-builder/_builds/microshift-installer.${ARCH}.iso` bootable RHEL for Edge OS image.
 
+### Disk Partitioning
+The `kickstart.ks` file is configured to partition the main disk using `Logical Volume Manager` (LVM). Such parititioning is required for the data volume used by the MicroShift CSI driver and it allows for flexible file system customization if the disk space runs out.
+
+By default, the following partition layout is created and formatted with the `XFS` file system:
+* Boot partition is allocated on a 1GB volume
+* The rest of the disk is managed by the `LVM`
+  * System root partition is allocated on a 5GB volume (minimal recommended size for a root partition)
+  * Data partition is allocated on the remaining disk space and mounted at `/var/mnt/data` directory
+
+> The swap partition is not created as it is not required by MicroShift.
```

Comment:
For this development image, we may want to avoid setting up SWAP to detect problems related to memory pressure. I will add a commented out line to the template that can be used for SWAP creation.